frontend/src/components/auth/StudentLogin.tsx

```diff
@@ -0,0 +1,118 @@
+import React, { useState } from "react";
+import { useHistory } from "react-router-dom";
+import { PinInput, PinInputField, HStack, Button } from "@chakra-ui/react";
+import AuthWrapper from "./AuthWrapper";
+import { STUDENT_SIGNUP_IMAGE } from "../../assets/images";
+import { LeftArrowIcon } from "../../assets/icons";
+
+const StudentLogin = (): React.ReactElement => {
+  // For back button
+  const history = useHistory();
+
+  // Some CS
+  const pinWidth = "60px";
+  const pinHeight = "100px";
+  const margin = "10px 10px 90px 10px !important";
```

Comment:
for CSS like width and height, let's use relative measurements where possible! this is to better support different screen sizes. you can read more about this here: https://medium.com/@madhum86/css-font-sizing-pixels-vs-em-vs-rem-vs-percent-vs-viewport-units-b1485716afe7